server/command.go

```diff
@@ -179,6 +197,22 @@ func (p *Plugin) executeLinkCommand(c *plugin.Context, args *model.CommandArgs,
 }
 
 func (p *Plugin) executeUnlinkCommand(c *plugin.Context, args *model.CommandArgs) (*model.CommandResponse, *model.AppError) {
+	isSysAdmin, err := p.IsSystemAdmin(args.UserId)
+	if err != nil {
+		return p.cmdError(args.UserId, args.ChannelId, "Something went wrong.")
+	}
+
+	if !isSysAdmin {
+		isChannelAdmin, err := p.IsChannelAdmin(args.UserId, args.ChannelId)
```

Comment:
i think we can fix lint errors coming from our code. Like here we have shadow declaration for err